frontend/static/keycloak.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
This file can't get committed.

Reply:
yes.  didn't mean to include that one.  will fix.